server/views/pages/statementDetailMacro.njk

```diff
@@ -1,68 +1,78 @@
+    </p>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-third">
+    <p class="govuk-!-margin-bottom-0 govuk-label--s">Date and time of incident</p>
+    <p class="govuk-!-margin-top-0" data-qa="date-and-time">
+      {{ data.incidentDate | formatDate('D MMMM YYYY - HH:mm')  }}
+    </p>
+  </div>
+  <div class="govuk-grid-column-one-third">
+    <p class="govuk-!-margin-bottom-0 govuk-label--s">Year joined the prison service</p>
+    <p class="govuk-!-margin-top-0" data-qa="job-start-year">
+      {{ data.jobStartYear }}
+    </p>
+  </div>
+</div>
+
+<div class="govuk-grid-column-two-thirds govuk-!-padding-0">
+  <hr class="govuk-!-margin-top-4" />
+</div>
+
+<div class="govuk-grid-row govuk-!-margin-top-4">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-!-margin-bottom-0 govuk-label--s">
+      Statement submitted
+    </p>
+    <p class="govuk-!-margin-top-0">
+      {% if data.submittedDate %}
+      {{ data.submittedDate | formatDate('D MMMM YYYY - HH:mm') }}
+    </p>
     {% endif %}
   </div>
-  
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <p class="govuk-!-margin-bottom-1 govuk-label--xs statement" data-qa="statement">{{ data.statement }}</p>
-    </div>
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-!-margin-bottom-1 govuk-label--xs statement" data-qa="statement">{{ data.statement }}</p>
   </div>
-  <hr />
 
-  {% for comment in data.additionalComments %}
-  <div class="govuk-grid-row govuk-!-margin-top-4">
-    <div class="govuk-grid-column-one-half">
-      <p class="govuk-!-margin-bottom-0 govuk-label--s">Additional comment</p>
-    </div>
-    <div class="govuk-grid-column-one-half ">
-      <p>
-        <span class="govuk-label--s">Submitted:</span> {{ comment.dateSubmitted | formatDate('D MMMM YYYY - HH:mm') }}
-      </p>
-    </div>
+  <div class="govuk-grid-column-two-thirds">
+    <hr class="govuk-!-margin-top-6" />
+  </div>
+</div>
+
+
+
+{% for comment in data.additionalComments %}
+<div class="govuk-grid-row govuk-!-margin-top-4">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-!-margin-bottom-0 govuk-label--s">Additional comment</p>
+  </div>
+
+  <div class="govuk-grid-column-one-half ">
+    <p> <span class="govuk-label--s">Submitted:</span> {{ comment.dateSubmitted | formatDate('D MMMM YYYY - HH:mm') }}</p>
+  </div>
   </div>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <p class="govuk-!-margin-bottom-1 govuk-!-margin-top-0 govuk-label--xs statement" data-loop={{loop.index}}
-        data-qa="viewAdditionalComment">{{comment.additionalComment}}</p>
-    </div>
+        data-qa="viewAdditionalComment">{{comment.additionalComment}}
+      </p>
   </div>
-  <hr />
-  {% endfor %}
+  <div class="govuk-grid-column-two-thirds">
+    <hr class="govuk-!-margin-top-6" />
+  </div>
```

Comment:
is it a bit weird having divs around the hrs? could they not be included as part of the previous div?

Reply:
Agreed. Changes made. 
Also moved date for the additional conditions - as discussed